nvm.sh

```diff
@@ -1586,10 +1586,16 @@ nvm_install_binary() {
   local TMPDIR
   local VERSION_PATH
 
+  local PROGRESS_BAR
   local NODE_OR_IOJS
   if [ "${FLAVOR}" = 'node' ]; then
     NODE_OR_IOJS="${FLAVOR}"
   fi
+  if [ "${NVM_NO_PROGRESS}" = "1" ]; then
```

Comment:
For things like this it's best to test if the environment variable is non-zero length:

```bash
if [ -n "${NVM_NO_PROGRESS}" ]; then
...
fi
```

Either it exists and has content or doesn't is how I've seen it typically used in other projects which are bash-based (of this kind of style of testing).  I tend to like that style of testing more.